pennylane_qiskit/converter.py

```diff
@@ -36,17 +36,19 @@
 inv_map = {v.__name__: k for k, v in QISKIT_OPERATION_MAP.items()}
 
 
-def _check_parameter_bound(param):
+def _check_parameter_bound(param, var_ref_map):
     """Utility function determining if a certain parameter in a QuantumCircuit has
     been bound.
 
     Args:
-        param: the parameter to be checked
+        param (qiskit.circuit.Parameter): the parameter to be checked
+        var_ref_map (dict[qiskit.circuit.Parameter, pennylane.variable.VariableRef]):
```

Comment:
At this point, it might be worth considering the `typing` module:

```python
from typing import Dict

def _check_parameter_bound(param: ..., var_ref_map: Dict[Parameter, VariableRef]) -> Parameter:
    """
    ...
    var_ref_map (Dict[Parameter, VariableRef]): ...
    """
    ...
```

Reply:
Good idea, I will add that in!